wled00/wled.cpp

```diff
@@ -979,6 +974,7 @@ void WLED::handleConnection()
       initConnection();
       interfacesInited = false;
       scanDone = true;
+      return;
```

Comment:
Does this return actually fix the issue? I'm unclear if this PR is a fix or just adding debug info to help try and find the issue so we can then fix @blazoncek

Reply:
Yes, this is the actual fix.
Debug changes were added to find the root cause.

Reply:
Great :)

Are you confident enough in this fix for this to go straight out into the GA release or do we need to get some feedback from the dev branch first?

Reply:
I hunted this for a month. 😄 And the multiwifi implementation is mine.
The bug is odd. When connecting to WiFi after boot if 1st SSID is accessible it will connect to it but subsequent attempts (if disconnected) to it will never commence and only 2nd or 3rd SSID will be tried.
This early return insures that also 1st SSID is correctly retried.